### PR TITLE
[REMOTE-WDPT-0602] - Pedro Lamy

### PR DESCRIPTION
**Pull request by PedroMMLamy**
Not much time for iteration 3.2.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -23,3 +23,24 @@ app.get('/', (req, res) => {
 });
 
 app.listen(3000, () => console.log('🏃‍ on port 3000'));
```
Comment (valmgisbert) [inline app.js:25]: Be sure to check which port you're using and clean your app.js when you do. Was there a specific reason why your port 3000 wasn't working?

**Comment by PedroMMLamy** [comment]
Val I have no idea why my port wasn't working

A terça, 28/07/2020, 18:39, valmgisbert <notifications@github.com> escreveu:

File: app.js [hunk 2]
```diff
@@ -23,3 +23,24 @@ app.get('/', (req, res) => {
...
+app.get('/beers', (req, res) => {
+  const beers = punkAPI.getBeers()
+
+  beers.then(beersArr => {
+    let newArr = beersArr
```
Comment (valmgisbert) [inline app.js:30]: API calls usually are pretty direct in what they do, but if it works for you to store it for better understanding be careful with how you name it; be as specific as you can to have it structured and easy to read.

File: app.js [hunk 3]
```diff
@@ -23,3 +23,24 @@ app.get('/', (req, res) => {
...
+
+});
+
+app.get('/random-beers', (req, res) => {
+  const beer = punkAPI.getRandom()
+
+  beer.then(resApi => {
+    let randoBeer = resApi
+    res.render('random-beers', {randoBeer}) })
+  .catch(error => console.log(error));
```
Comment (valmgisbert) [inline app.js:42]: work on your spacing a bit to read better your code

**Comment by valmgisbert** [comment]
Your app.js is good! keep in mind the comments, and whenever you can please finish working on this lab.

**Pull request closed without merging**